Sources/Views/InputBarItem.swift

```diff
@@ -166,90 +191,152 @@ open class InputBarButtonItem: UIButton {
     
     // MARK: - Hook Setup Methods
     
+    /// Used to setup your own initial properties
+    ///
+    /// - Parameter item: A reference to Self
+    /// - Returns: Self
     @discardableResult
-    open func configure(_ setup: InputBarButtonItemAction) -> Self {
-        setup(self)
+    open func configure(_ item: InputBarButtonItemAction) -> Self {
```

Comment:
This is an API breaking change 😅

Reply:
Is it? Because of the _ no one should have already enforced the label to be 'setup'

Reply:
You're right! I overlooked the parameter name omission

Reply:
Cool, i just figured setup was a poor descriptor